Fails for me with xcode7 on 10.11:
```
server-tools/instance-manager/Makefile.am:103: `%'-style pattern rules are a GNU make extension
/usr/bin/clang -I. -fheinous-gnu-extensions -no-cpp-precomp -arch x86_64 -mmacosx-version-min=10.8  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -isystem /Users/Shared/xbm
c-depends/macosx10.11_x86_64-target/include  -DHAVE_CONFIG_H -fheinous-gnu-extensions -no-cpp-precomp -arch x86_64 -mmacosx-version-min=10.8  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/
MacOSX10.11.sdk -isystem /Users/Shared/xbmc-depends/macosx10.11_x86_64-target/include  -ggdb3 -Wall -W   -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes   -Wpointer-arith -Wbad-function-cast -Wnested-externs -fPIC -MT ne
ttle-internal.o -MD -MP -MF nettle-internal.o.d -c nettle-internal.c \
        && true
../lib/replace/replace.c:811:23: error: unknown type name 'clockid_t'; did you mean 'clock_t'?
int rep_clock_gettime(clockid_t clk_id, struct timespec *tp)
                      ^~~~~~~~~
                      clock_t
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/sys/_types/_clock_t.h:30:33: note: 'clock_t' declared here
typedef __darwin_clock_t        clock_t;
                                ^
1 error generated.
```

--- a/docs/README.osx
+++ b/docs/README.osx
@@ -32,7 +32,8 @@ codecs that support a multitude of music and video formats.
 
 On Mavericks (OSX 10.9.x) we recommend using Xcode 6.1.
 On Yosemite (OSX 10.10.x) we recommend using Xcode 6.4.
-On El Capitan (OSX 10.11.x) we recommend using Xcode 7.2.
+On El Capitan (OSX 10.11.x) we recommend using Xcode 7.x or Xcode 8.x.
+On Sierra (macOS 10.12.x) we recomment using Xcode 8.x.
 
 NOTE TO NEW OS X USERS: All lines that are prefixed with the '$' character are
 commands that need to be typed into a Terminal window. Note that the '$'
@@ -69,6 +70,8 @@ constellations of Xcode and osx versions (to be updated once we know more):
 4. XCode 6.3.0 against OSX SDK 10.10 (Y)
 5. Xcode 6.4.0 against OSX SDK 10.10 (Y)
 6. Xcode 7.x against OSX SDK 10.11 (EC)
+7. Xcode 8.0 against OSX SDK 10.12 (EC)
+8. Xcode 8.0 against OSX SDK 10.12 (S)
 
 -----------------------------------------------------------------------------
 3.2 Install Kodi build depends

--- a/tools/depends/target/samba-gplv3/Makefile
+++ b/tools/depends/target/samba-gplv3/Makefile
@@ -52,6 +52,9 @@ endif
 ifeq ($(TARGET_PLATFORM),appletvos)
 	cd $(PLATFORM); patch -p0 < ../no_fork_and_exec.patch
 endif
+ifeq ($(OS),osx)
+	sed -ie "s|ifndef HAVE_CLOCK_GETTIME|if !defined(HAVE_CLOCK_GETTIME) \&\& !defined(CLOCK_REALTIME)|" "$(PLATFORM)/lib/replace/system/time.h"
+endif
 	cd $(PLATFORM)/source3; $(CONFIGURE)
 
 $(LIBDYLIB): $(PLATFORM)

